crates/sui-indexer/migrations/mysql/2024-05-05-155158_obj_indices/down.sql

```diff
@@ -0,0 +1,2 @@
+-- This file should undo anything in `up.sql`
+-- TODO: objects_version
```

Comment:
I left these blank for now, but let me know what the best way to handle this is @sadhansood (whether I should fill this out or leave it blank, what the best way is to test it, etc).

I noticed that the Indexer tests don't run these migrations (at least on the Postgres side, I have to run the GraphQL E2E tests to test this kind of code), so I'm nervous to touch this without knowing how to make sure it's working.